README.md

```diff
@@ -1,3 +1,127 @@
-[API Repo Linke](https://github.com/GreyMatteOr/bangwords-api) 
+[![Contributors][contributors-shield]](https://github.com/GreyMatteOr/bangwords/graphs/contributors)
+[![Forks][forks-shield]](https://github.com/GreyMatteOr/bangwords/network/members)
+[![Issues][issues-shield]](https://github.com/GreyMatteOr/bangwords/issues)
+[![Stars][stars-shield]](https://github.com/GreyMatteOr/bangwords)
 
- [API](https://bangwords-api.herokuapp.com/) 
+## BangWords (a shameless ripoff)
```

Comment:
I love the humor, but I think it's a little un-professional. Let's be proud of what we made and even if we all know the truth about what it is, let's try our best to represent it well.

And besides, I think as part of our stretches it will end up much different than the original.